packages/cms/pages/settings/team-members/team-members.tsx

```diff
@@ -117,24 +135,60 @@ export const TeamMembers: FunctionComponent<ProfileProps> = () => {
         actions: [
           {
             name: 'Edit',
-            description: 'Edit this item',
+            description: 'Change user role',
             icon: 'pencil',
             type: 'icon',
             onClick: item => {}
           },
           {
             name: 'Delete',
-            description: 'Delete this item',
+            description: 'Remove this user',
             icon: 'trash',
             type: 'icon',
             color: 'danger',
-            onClick: item => {}
+            disabled: true,
+            onClick: item => {
+              if (
+                item.adminRoles.some(
```

Comment:
I've seen this snippet of code like twice now. Maybe move it to a small helper function?

Reply:
okay